src/utils.ts

```diff
@@ -151,3 +198,31 @@ export function parseYamlFromText(body: string): any {
 export function parseBodyFromText(body: string): string {
   return body.replace(/.*---\n[^)]*---\n.*/g, '')
 }
+
+/**
+ * get a unique list
+ * @param {string[]} the array to make unique
+ * @returns {string[]} the unique list
+ */
+export function unique(input: string[]): string[] {
+  return input.filter((item, i, ar) => ar.indexOf(item) === i)
+}
```

Comment:
I don't know if this function is worth having. It's only used in one place and it's probably better to use a Set constructor do make the list unique.

What do you think @Chocrates @jasonmacgowan ?

Reply:
If `Set` does what we want then thats less code to maintain.

Reply:
Using the `new Set()` is looking good, thanks @froi for this recommendation! marking it resolved.